src/test/java/org/beanpod/switchboard/service/UserServiceTest.java

```diff
@@ -0,0 +1,67 @@
+package org.beanpod.switchboard.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.util.Optional;
+import org.beanpod.switchboard.entity.UserEntity;
+import org.beanpod.switchboard.fixture.UserFixture;
+import org.beanpod.switchboard.repository.UserRepository;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+
+class UserServiceTest {
+
+  @Mock private static UserEntity userEntity;
```

Comment:
I think this variable is unused in the tests

Reply:
used in line 56

Reply:
fixed, removed